packages/botonic-react/src/webchat-app.tsx

```diff
@@ -248,21 +187,35 @@ export class WebchatApp {
     this.onClose && this.onClose(this, ...args)
   }
 
-  async onUserInput({ user, input }) {
+  async onUserInput({ user, input }: OnUserInputArgs): Promise<void> {
+    if (!user) return
+
     this.onMessage &&
       this.onMessage(this, {
         ...input,
         sentBy: SENDERS.user,
         isUnread: false,
-      })
-    return this.hubtypeService.postMessage(user, input)
+      } as unknown as WebchatMessage)
+    this.hubtypeService.postMessage(user, input)
+    return
```

Comment:
optional: if it is returning a void promise, I would leave it as it was

Reply:
It is the same and as it was before, the linter was complaining